python/paddle/fluid/layer_helper.py

```diff
@@ -151,13 +151,7 @@ def append_activation(self, input_var):
             act['use_mkldnn'] = self.kwargs.get('use_mkldnn')
         act_type = act.pop('type')
 
-        tmp = input_var
-        # NOTE(dzhwinter): some activation support inplace compution.
-        # NOTE(minqiyang): currently, we don't support inplace in imperative mode
-        if not _in_imperative_mode() and core.IsInplace(act_type):
-            tmp = input_var
-        else:
-            tmp = self.create_variable_for_type_inference(dtype=input_var.dtype)
+        tmp = self.create_variable_for_type_inference(dtype=input_var.dtype)
```

Comment:
why change this? we don't support inplace? @velconia

Reply:
> why change this? we don't support inplace? @velconia

same with above